src/bci_build/package/appcontainers.py

```diff
@@ -556,27 +555,26 @@ def _get_nginx_kwargs(os_version: OsVersion):
 TOMCAT_CONTAINERS = [
     ApplicationStackContainer(
         name="tomcat",
-        pretty_name=f"Apache Tomcat {tomcat_major}",
-        package_name=f"tomcat-{tomcat_major}-image",
+        pretty_name=f"Apache Tomcat {tomcat_major} with OpenJDK {jre_version}",
+        package_name=f"tomcat-{tomcat_major}-openjdk-{jre_version}-image",
         os_version=os_version,
         is_latest=(
             (os_version in CAN_BE_LATEST_OS_VERSION)
             and tomcat_major == _TOMCAT_VERSIONS[-1]
         ),
-        version=tomcat_major,
-        additional_versions=["%%tomcat_version%%", "%%tomcat_minor%%"],
+        version=f"{tomcat_major}-openjdk{jre_version}",
+        additional_versions=[
+            f"%%tomcat_version%%-openjdk{jre_version}",
+            f"%%tomcat_minor%%-openjdk{jre_version}",
+        ],
```

Comment:
This leads to tags of the form: `9-openjdk21` or `10-openjdk22`, which are nothing like we provide atm and don't really work too well with semver. I'd much rather have `tomcat-openjdk22:10` instead of `tomcat:10-openjdk22`

Reply:
why is "semver" a constraint?